jobQueue.py

```diff
@@ -79,7 +79,7 @@ def add(self, job):
             self.log.info("add|JobQueue is full")
             return -1
         self.log.debug("add|Gotten next ID: " + str(job.id))
-        self.log.info("add|Unassigning job %s" % str(job.id))
+        self.log.info("add|Unassigning job ID: %d" % (job.id))
```

Comment:
you're right. what I meant to say above was that the log messages elsewhere tend to have this established format of using `job ID: #` and this change was intended to formalize that.